blog/2022/01/22/o2-orion-monopoles.mdx

```diff
@@ -0,0 +1,173 @@
+suburban United Kingdom.
+
+# Introduction to Orion
+
+In 2021, O2 launched its **Beacon 3** network upgrade project, including its brand new, flagship Orion monopoles.
+
+<FactBox>
+
+"Beacon" was the codename for a series of joint upgrade plans made between O2 and Vodafone to provide LTE to the whole of the UK.
+
+Beacon 3 is the first Beacon project to include installation of <Tooltip text="one network operator">unilateral</Tooltip> sites as opposed to the
+multi-operator radio access network (MORAN) sites we typically see with Cornerstone Networks.
+
+This indicates a possibly deteriorating relationship between the two operators.
+
+</FactBox>
+
+Orion monopoles manage to cram a multitude of 4G bands, along with n28 and n78 5G and their standard 2G/3G all into **one street cabinet and one
+monopole**, often replacing what used to be basic 4G/3G/2G monopoles and several street cabinets.
+
+O2 have begun favouring these new deployments because of their simplicity. A **pole swap can be done in just one day**, consisting of contractors
+installing the feeders in the new cabinet, and an HIAB truck arriving with the new monopole, lifting and dropping it into place, and then taking
+the old one away to be stripped for parts.
+
+![An O2 Orion monopole alongside a previous Beacon 2 monopole](./orion_vs_old.jpg)
+
+# Planning applications
+
+For mobile networks to install new street infrastructure, they do not need to apply for planning permission. Instead, they must submit a notice
+of **prior notification**. This is a notice of intent, basically saying "I want to do something. Tell me if I can't.".
+
+The local planning authority has **56 days to object to the plans, or approve them**. No response after 56 days is classed as an approval. The
+big kick against conspiracy theorists is that **LPAs may only reject proposed development due to its siting or appearance**.
+
+The compact nature of Orion monopole sites mean that councils are more likely to approve them compared to monopoles from other operators which
+may clutter public areas with various street furniture.
+
+## Exemptions
+
+Exemptions exist from submitting prior notification plans, aimed to make minor changes easier for network operators. While they are meant for
+swapping out equipment when it fails or for adding new microwave link dishes to sites, O2 have now been seen using these powers for performing
+full-on Orion site swaps in some areas.
+
+One of my local monopole sites in Burgess Hill, for example, has recently been swapped from a Beacon 2 site to a new dual operator Orion with no
+prior notice submitted. This was possible due to one exemption:
+
+> ...prior approval does not apply to ground-based masts which when altered or replaced would not exceed the greater of the height of the
+> existing mast and 20m above ground level.
+
+Since my old local monopole was 20m in height, and so was the new Orion monopole, **the site could be swapped out without any form of approval
```

Comment:
```suggestion
Since my old local monopole was 20m in height, and so was the new Orion monopole, the site could be swapped out **without any form of approval required**.
```